app/views/directives/edit-environment-from.html

```diff
@@ -50,27 +80,40 @@
         </div>
       </div>
 
-      <div ng-if="!$ctrl.isEnvFromReadonly(entry) && $ctrl.hasEntries()" class="environment-from-editor-button">
-        <span
-          ng-if="!$ctrl.cannotSort && $ctrl.entries.length > 1"
-          class="fa fa-bars sort-row"
-          role="button"
-          aria-label="Move row"
-          aria-grabbed="false"
-          as-sortable-item-handle></span>
-        <a
-          ng-if="!$ctrl.cannotDeleteAny"
-          href=""
-          class="pficon pficon-close delete-row as-sortable-item-delete"
-          role="button"
-          aria-label="Delete row"
-          ng-click="$ctrl.deleteEntry($index, 1)"></a>
-      </div>
-      <div class="environment-from-view-details">
-        <a
-          href=""
-          ng-if="entry.selectedEnvFrom"
-          ng-click="$ctrl.viewOverlayPanel(entry.selectedEnvFrom)">View Details</a>
+      <div class="form-group environment-from-input">
+        <div class="environment-from-input" ng-if="!$ctrl.isEnvFromReadonly(entry) && $ctrl.hasOptions()">
+          <label for="envfrom-prefix-{{$index}}" class="sr-only">Prefix</label>
+          <input type="text"
```

Comment:
This should have a pattern with a corresponding validation error. (I'll need to look up the correct regex for prefix.)

Reply:
That works, yeah send it over